batch_slac/run_batch.py

```diff
@@ -23,30 +28,53 @@
 ## muon
 generate_muon   = False
 muon_x_range    = [-20.0,20.0]
-# muon_a_range    = [-3*1e-3, 3*1e-3]
-do_cov_angle    = True
-muon_a_range    = []
-
-out_loc         = here_batch+f"/out_files/{detcard_name}_bkgr_{bkg_rate}"
+muon_a_range    = [-3*1e-3, 3*1e-3]
+do_cov_angle    = False
+#muon_a_range    = []
+
+import subprocess
+result = subprocess.run(['hostname', '-d'], stdout=subprocess.PIPE)
+domain = result.stdout.decode('utf-8').split('\n')[0]
+if 'cern' in domain:
+#    out_loc         = script_dir +f"/out_files/{detcard_name}_bkgr_{bkg_rate}"
+    out_loc         =f"/Data/ML/si-mu-lator/simulation_data/{detcard_name}_bkgr_{bkg_rate}"
+else:
+    out_loc         = script_dir+f"/out_files/{detcard_name}_bkgr_{bkg_rate}"
 
 if not generate_muon and override_total_n_noise > 0:
-    out_loc += f'_override_{override_total_n_noise}/'
+    out_loc += f'_override_{override_total_n_noise}'
```

Comment:
Why change the behavior here?

Reply:
below at line 95 you add the slash /, which is natural when writing directories path.
I was just avoiding double slash //.
In fact the else below is not necessary in this way